geometry_msgs/Quaternion m_headRotation
geometry_msgs/Vector3    m_headTranslation

float32 m_eyeGazeLeftPitch
float32 m_eyeGazeLeftYaw
float32 m_eyeGazeRightPitch
float32 m_eyeGazeRightYaw

float32[] m_coeffs
